extensions/qbittorrent/src/menubar.tsx

```diff
@@ -0,0 +1,35 @@
+import { MenuBarExtra, Icon, Keyboard, openExtensionPreferences, launchCommand, LaunchType } from "@raycast/api";
+
+export default function Menubar() {
+  const showTorrents = () => launchCommand({ name: "torrents", type: LaunchType.UserInitiated });
+
+  const handleAddTorrent = () => launchCommand({ name: "add-torrents", type: LaunchType.UserInitiated });
```

Comment:
You might need to handle this as an async function and have it inside a try {} catch {} since it's possible to disable the commands and that will throw an error 🙂